.npmignore

```diff
@@ -0,0 +1,6 @@
+/artifacts
+/build
```

Comment:
don't we need the `build` dir to **not** be ignored so that yarn can pick it up?

Reply:
You’re right.

Reply:
Let's give this a spin locally to make sure it works before merging.

Reply:
It was working all along. Turns out that the `.npmignore` file is not needed because we specifiy `files` explicitly in `package.json`.

Reply:
even better :+1: